bcl_manifest.json

```diff
@@ -55,5 +55,10 @@
     "org": "Ski90Moo",
     "type": "component",
     "url": "https://github.com/Ski90Moo/OpenStudio_Components_HEPLLC"
+  },{
+    "name": "OpenStudio Occupant-related Variability Measures Gem",
```

Comment:
```suggestion
    "name": "OpenStudio-Occupant-Variability-Gem",
```

Reply:
the name must match your repo name exactly.  otherwise it looks good!

Reply:
Thanks @kflemin. I updated it to match the repo name.